doc/multi-o.md

```diff
@@ -0,0 +1,117 @@
+    -ethAcctAddr <ETHEREUM_ADDRESS> \
+    -ethPassword <ETHEREUM_ACCOUNT_PASSWORD (string|path)> \
+    -ethController <CONTROLLER_CONTRACT_ADDRESS (required if '-network' not provided)>
+```
+
+## Single Orchestrator with Redeemer
+
+This allows an orchestrator running a single on-chain node to use a seperate Ethereum account to redeem winning tickets on-chain and pay for that transaction while the recipient is still orchestrator node's Ethereum address.
+
+The orchestrator node will still be responsible for initialising rounds and calling `reward`. 
+
+1. Start the Redeemer 
+
+```shell
+livepeer
+    <...ETH SETUP...\
+    -redeemer=true \ 
+    -httpAddr <REDEEMER_HTTP_ADDR (host):port \ 
+    -ethOrchAddr <ORCHESTRATOR_ON_CHAIN_ETH_ADDR (also the recipient address)>
+```
+
+2. Start the Orchestrator
+
+
+```shell
+livepeer
+    <...ETH SETUP...> \
+    -orchestrator=true -transcoder=true \
+    -initializeRound=true \
+    -httpAddr <ORCH_HTTP_ADDR (host):port \
+    -pricePerUnit <PRICE (wei/pixel if '-pixelsPerUnit' is not set)
+```
+
+
+## Blockchain Service Node with Multiple Orchestrators
+
+In this setup a node started with the keys for the on-chain registered address will be responsible for all transactions. 
+
+In order to use multiple orchestrators with this setup, the cluster of orchestrator nodes responsible for transcoding would have to be behind a load balancer, **the URI of the load balancer will be the on-chain registered Service URI.**
+
+1. Start the Blockchain Service Node 
+
+```shell
+livepeer
+    <...ETH SETUP...\
+    -orchestrator= true \ 
+    -redeemer=true \ 
+    -httpAddr <REDEEMER_HTTP_ADDR (host):port \
+    -initializeRound=true 
+```
+
+2. Start an Orchestrator 
+```shell
+livepeer
+    <...ETH SETUP...> \
+    -orchestrator=true -transcoder=true \
+    -ethOrchAddr <ORCHESTRATOR_ON_CHAIN_ETH_ADDR (also the recipient address)> \
+    -redeemerAddr <REDEEMER_HTTP_ADDR> \
+    -httpAddr <ORCH_HTTP_ADDR (host):port \
+    -pricePerUnit <PRICE (wei/pixel if '-pixelsPerUnit' is not set)
+```
+
+## Redeemer + RewardService & RoundInitializer + Multiple Orchestrators
+
+This is the "coldest" setup possible for on-chain registered addresses. The keys for the on-chain registered address will be responsible for initialising rounds and calling reward, since reward can not be called on behalf of another address in the current version of the protocol.
+
+To use this setup with multiple orchestrators a load balancer is required as described above. 
+
+1. Start the Redeemer
+
+```shell
+livepeer
+    <...ETH SETUP...\
+    -redeemer=true \ 
+    -httpAddr <REDEEMER_HTTP_ADDR (host):port \ 
+    -ethOrchAddr <ORCHESTRATOR_ON_CHAIN_ETH_ADDR (also the recipient address)>
+```
+
+2. Start the RewardService & RoundInitializer 
+
+In the current go-livepeer version (0.5.8) this node type will still run a transcoder server. To avoid this node also transcoding avoid making it accessible through the load balancer of your choice. 
```

Comment:
This isn't really a documentation issue per se, but just noting that reading through this makes me realize that we really need to loosen up restriction on who can call `reward`, because that would simplify much of the guidance here.

> To avoid this node also transcoding avoid making it accessible through the load balancer of your choice.

An easier way may be to set the ServiceURI to localhost and ensure the box itself is not available to the public?

"To prevent this node from transcoding, set the service URI to localhost and ensure the system is not behind a publicly available IP: `-serviceAddr 127.0.0.1:8935`"

(BTW I just checked this and it seems that we listen to all ipv6 interfaces regardless of what the ipv4 address is, so that needs to be corrected at some point, but should be a non-issue as long as the system itself isn't open to the public, eg NAT'd.)

Reply:
>  we really need to loosen up restriction on who can call reward

Yeah we briefly discussed this on discord yesterday. This would be a requirement indeed before we can have complete cold storage for on-chain registered addresses. 

> An easier way may be to set the ServiceURI to localhost and ensure the box itself is not available to the public?

This only overwrites the serviceURI locally and not on-chain, right?

Reply:
> This only overwrites the serviceURI locally and not on-chain, right?

Correct. The user should also ensure the on-chain address points elsewhere, and that the current host not publicly available.

Reply:
This should no longer be necessary with the latest changes in #1547